modules/claims_api/app/services/claims_api/dependent_claimant_poa_assignment_service.rb

```diff
@@ -101,15 +101,15 @@ def build_benefit_claim_update_input(claim_details:)
     end
 
     def assign_poa_to_dependent_via_update_benefit_claim?
-      first_open_claim_details
+      claim_details = first_open_claim_details
 
-      if first_open_claim_details.blank?
+      if claim_details.blank?
         log(detail: 'Dependent has no open claims.', statuses: dependent_claims.pluck(:phase_type).uniq)
 
         raise ::Common::Exceptions::ServiceError
       end
 
-      benefit_claim_update_input = build_benefit_claim_update_input(claim_details: first_open_claim_details)
+      benefit_claim_update_input = build_benefit_claim_update_input(claim_details:)
```

Comment:
added this variable because before `first_open_claim_details` was being called 3 times independently